src/graphql/event.rs

```diff
@@ -944,9 +943,9 @@ fn convert_sensors(
             bail!("no such sensor")
         };
 
-        if let Some(node_settings) = node.settings {
-            if !node_settings.hostname.is_empty() {
-                converted_sensors.push(node_settings.hostname.clone());
+        if let Some(node_profile) = node.0.profile {
+            if !node_profile.hostname.is_empty() {
+                converted_sensors.push(node_profile.hostname.clone());
```

Comment:
A test seems required here to pass Codecov.

Reply:
Because Control.rs is going to change, it doesn't need to pass Codecov for now. However, I think the other code related to this PR should.